Sources/Sentry/SentryReplayApi.m

```diff
@@ -22,18 +23,49 @@ - (void)unmaskView:(UIView *)view
 
 - (void)pause
 {
-    SentrySessionReplayIntegration *replayIntegration =
-        [SentrySDK.currentHub getInstalledIntegration:SentrySessionReplayIntegration.class];
+    SentrySessionReplayIntegration *replayIntegration
+        = (SentrySessionReplayIntegration *)[SentrySDK.currentHub
+            getInstalledIntegration:SentrySessionReplayIntegration.class];
     [replayIntegration pause];
 }
 
 - (void)resume
 {
-    SentrySessionReplayIntegration *replayIntegration =
-        [SentrySDK.currentHub getInstalledIntegration:SentrySessionReplayIntegration.class];
+    SentrySessionReplayIntegration *replayIntegration
+        = (SentrySessionReplayIntegration *)[SentrySDK.currentHub
+            getInstalledIntegration:SentrySessionReplayIntegration.class];
     [replayIntegration resume];
 }
 
+- (void)start
+{
+    SentrySessionReplayIntegration *replayIntegration
+        = (SentrySessionReplayIntegration *)[SentrySDK.currentHub
+            getInstalledIntegration:SentrySessionReplayIntegration.class];
+
+    if (replayIntegration == nil) {
+        replayIntegration = [[SentrySessionReplayIntegration alloc] init];
+
+        SentryOptions *options = [[SentryOptions alloc] init];
+        options.enableSwizzling = SentrySDK.currentHub.client.options.enableSwizzling;
+        options.experimental.sessionReplay.sessionSampleRate = 1;
```

Comment:
`h`: What if you want to sample your replays when using the start/stop API? I expect that the SDK uses the sample rate of the options even with the manual API.

Reply:
as a counterpoint, the continuous profiler does not respect any sample rate when using the start/stop calls, and we document that if customers want sampling, they need to handle it (arguable on whether we should offer it, but that's just how it is right now. maybe sample rates only make sense for the automatic integration stuff)

Reply:
> What if you want to sample your replays when using the start/stop API? I expect that the SDK uses the sample rate of the options even with the manual API.

Not an option, if the user call start it means they want to replay running for sure.

Reply:
If that's intended, then all good. Just wanted to double-ckeck.